src/frontend/src/pages/ChangeRequestDetailPage/ProposedSolutionsList.tsx

```diff
@@ -48,9 +48,10 @@ const ProposedSolutionsList: React.FC<ProposedSolutionsListProps> = ({ proposedS
   return (
     <>
       <div className={styles.proposedSolutionsList}>
-        {proposedSolutions.map((proposedSolution, i) => (
-          <ProposedSolutionView key={i} proposedSolution={proposedSolution} />
-        ))}
+        {proposedSolutions.map((proposedSolution, i) => {
```

Comment:
You should undo the changes in this file.

Reply:
fixed in [c69b8ff](https://github.com/Northeastern-Electric-Racing/FinishLine/pull/611/commits/c69b8ff1dc462fd5bc0f367c08f0719ddd11cb0b)